src/components/PrivateRoute/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Navigate } from 'react-router';
+
+interface IPrivateRouteProps {
+    component: JSX.Element;
+}
+
+export const PrivateRoute = (props: IPrivateRouteProps) => {
```

Comment:
Можно разворачивать сразу в аргументе

```suggestion
export const PrivateRoute = ({component}: IPrivateRouteProps) => {
```